orc8r/cloud/docker/controller/supervisor_logger.py

```diff
@@ -42,18 +37,14 @@ def main():
         data = sys.stdin.read(int(headers['len']))
 
         # transition from READY to ACKNOWLEDGED
-        write_stdout('RESULT %s\n%s' % (len(data), data))
-
+        write_stdout('RESULT 2\nOK')
 
-def result_handler(event, response):
-    # Parse the headers
-    line, data = response.split('\n', 1)
-    headers = dict(x.split(':') for x in line.split())
 
-    # Get the log lines and prefix the process name and stdout/stderr
-    lines = data.rstrip().split('\n')
-    prefix = '%s %s | ' % (headers['processname'], headers['channel'])
-    print('\n'.join([prefix + l for l in lines]))
+def result_handler(event, _):
```

Comment:
Should this react the same as before? I don't see This at a glance - can someone elaborate?

Reply:
This was the most complicated part of the change.

In the old version, the log message in `data` was written to stdout in the main loop and then processed in the result handler as part of the response. After the update, supervisor will just stop logging anything if you do that because it violates the event listener protocol, see for example: https://github.com/Supervisor/supervisor/issues/1417

In the new version, I implemented a result handler that takes the log message from the log events. I think the only way to see that it behaves the same way is to compare the controller logs before and after the change.